query-engine/connector-test-kit-rs/query-tests-setup/src/runner/mod.rs

```diff
@@ -33,14 +34,18 @@ pub struct Runner {
 }
 
 impl Runner {
+    pub fn prisma_dml(&self) -> &str {
+        &self.prisma_dml
```

Comment:
Could this be replaced with something like (off the top of my head) `self.query_schema.internal_data_model.schema.db.source()`? That would avoid cloning a large string for every test.

Reply:
Will try to! Great suggestion!